### PR TITLE
added event and task classes

### PR DESCRIPTION
**Pull request by Dachi-Laperashvili**

### CODE REVIEW

File: src/main/java/com/example/eventorganizer/task/State.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package com.example.eventorganizer.task;
+public enum State {
```
Comment (giorobota) [inline src/main/java/com/example/eventorganizer/task/State.java]: Please rename it to TaskState so we know where this enum belongs

File: src/main/java/com/example/eventorganizer/task/Task.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package com.example.eventorganizer.task;
+
+import com.example.eventorganizer.event.Event;
+import com.example.eventorganizer.user.User;
+import jakarta.persistence.*;
+@Entity
```
Comment (giorobota) [inline src/main/java/com/example/eventorganizer/task/Task.java:6]: Add one line separation between imports and code

**Review by giorobota — CHANGES_REQUESTED** [review]
Good job, I left some small comments, please update your commit message according to conventional commits https://www.conventionalcommits.org/en/v1.0.0/

Also, let's include flyway migrations in this PR as well:
https://www.baeldung.com/database-migrations-with-flyway

File: src/main/resources/db/migration/V1__.sql [hunk 3]
```diff
@@ -0,0 +1,33 @@
+CREATE TABLE IF NOT EXISTS users
+(
+    id       BIGINT GENERATED BY DEFAULT AS IDENTITY NOT NULL,
+    email    VARCHAR(255),
+    password VARCHAR(255),
+    CONSTRAINT users_pkey PRIMARY KEY (id)
+
+);
+
+CREATE TABLE IF NOT EXISTS events
```
Comment (giorobota) [inline src/main/resources/db/migration/V1__.sql]: Please move each table creation to separate file

File: src/main/resources/application.properties [hunk 4]
```diff
@@ -1,7 +1,14 @@
...
 spring.jpa.show-sql=true
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
-spring.jpa.properties.hibernate.format_sql=true
\ No newline at end of file
+spring.jpa.properties.hibernate.format_sql=true
+
+spring.flyway.enabled=true
+spring.flyway.url=jdbc:postgresql://localhost:5432/event_organizer
+spring.flyway.user=dachi
+spring.flyway.password=password
```
Comment (giorobota) [inline src/main/resources/application.properties]: Passwords should not be visible in git repository. You may have also received github's security warning email.
You can pass passwords as environment variables, like 
`spring.flyway.password=${DB_PASSWORD}`
and run application with parameters DB_PASSWORD=password

**Review by giorobota — CHANGES_REQUESTED** [review]

File: src/main/resources/application.properties (hunk 4 above)
Comment (giorobota) [reply src/main/resources/application.properties]: https://www.jetbrains.com/help/objc/add-environment-variables-and-program-arguments.html

**Review by giorgi-robitashvili — APPROVED** [review]

**Review by giorgi-robitashvili — APPROVED** [review]

**Pull request merged**